### PR TITLE
Book Detail II

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Add a new image for the book detail card.

## Screenshots

image

## Trello Card

https://trello.com/c/v7Tat0Ew/27-detalle-de-libro-ii-imagen

## Blocked By

#9

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -9,12 +9,15 @@
...
-  <div class='book'>
-    <div class='book-title'><h1 class='title'>Título del Libro</h1> <p class='title-genre'>(género)</p></div>
-    <div class='book-details'>
-      <p class='book-item'><strong>Autor del Libro: </strong> <span class='book-item-span'>del autor del Libro</span></p>
-      <p class='book-item'><strong>Editorial: </strong><span class='book-item-span'>Nombre de editorial</span></p>
-      <p class='book-item'><strong>Año de Publicación: </strong><span class='book-item-span'>Año de Publicación</span></p>
+  <div class='book-card'>
+    <img class='book-image' src='assets/book-cover.png'>
+    <div class='book'>
+      <div class='book-title'><h1 class='title'>Título del Libro</h1> <p class='title-genre'>(género)</p></div>
```
Comment (erikankodelia) [inline maquetado/index.html]: Put in the next line the `h1` tag and the same with the tag `p`

File: maquetado/scss/details.scss [hunk 2]
```diff
@@ -26,3 +26,17 @@
...
   margin-right: 10px;
 }
+
+.book-card {
+  .book-image {
+    max-width: 261px;
+    max-height: 368px;
+    padding: 24px;
+  }
+  display: flex;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Put first these props and then the `.book-image`

File: maquetado/scss/details.scss [hunk 3]
```diff
@@ -26,3 +26,17 @@
...
+  flex-flow: row;
+  justify-content: flex-start;
+  max-width: 900px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: When you put max-width, add width: 100%

File: maquetado/scss/details.scss [hunk 4]
```diff
@@ -26,3 +26,17 @@
...
+  max-height: 416px;
+  margin: 5px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Why you need this margin?

File: maquetado/index.html (hunk 1 above)
Comment (mcolombo87) [reply maquetado/index.html]: ok!

File: maquetado/scss/details.scss (hunk 4 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: For the screenshot! jajaja. In the next PRs that margin was removed.

File: maquetado/scss/details.scss [hunk 5]
```diff
@@ -26,3 +27,22 @@
...
+
+.book-card {
+  display: flex;
+  flex-flow: row;
+  justify-content: flex-start;
+  max-width: 900px;
+  width:100%;
+  max-height: 416px;
+  height: 100%;
+  margin: 5px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Sort alphabetically the props inside the class

File: maquetado/scss/details.scss [hunk 6]
```diff
@@ -26,3 +27,22 @@
...
+
+  .book-image {
+    max-width: 261px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Same here

**Pull request merged**